### PR TITLE
[Keymap] planck layout for kifinnsson

### PR DESCRIPTION
**Pull request by kifinnsson**
## Description

My daily driver layout with placeholder cases in process_record_user() for my work macros. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/kifinnsson/keymap.c [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+
+bool     is_lgui_active = false;
+uint16_t lgui_timer     = 0;
+
+extern keymap_config_t keymap_config;
```
Comment (noroadsleft) [inline keyboards/planck/keymaps/kifinnsson/keymap.c]:
```suggestion
```

File: keyboards/planck/keymaps/kifinnsson/keymap.c [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+     * |      | Alt  | WH_D |Shift | Ctrl | MS_D | PgDn | Left | Down |Right | Bksp | Menu |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * | WH_L | Undo | Cut  | Copy |Paste |MS_B_1|MS_B_2|MS_B_3| MS_L | MS_R | WH_R |      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |      |      |      |      |      |
+     * `-----------------------------------------------------------------------------------'
+     */
+    [_EXTEND] = LAYOUT_planck_grid(
+        _______, KC_ESC,  KC_WH_U, KC_WBAK, KC_WFWD, KC_MS_U, KC_PGUP, KC_HOME, KC_UP,   KC_END,  KC_DEL,  KC_ESC,
+        _______, KC_LALT, KC_WH_D, KC_LSFT, KC_LCTL, KC_MS_D, KC_PGDN, KC_LEFT, KC_DOWN, KC_RGHT, KC_BSPC, KC_MENU,
```
Comment (noroadsleft) [inline keyboards/planck/keymaps/kifinnsson/keymap.c]: **NOTE**: If you're wanting the Context Menu key, its keycode is actually `KC_APP`.

```suggestion
        _______, KC_LALT, KC_WH_D, KC_LSFT, KC_LCTL, KC_MS_D, KC_PGDN, KC_LEFT, KC_DOWN, KC_RGHT, KC_BSPC, KC_APP,
```

**Comment by kifinnsson** [comment]
I'm sure there is it's mostly for my own readability to be honest. They
tend to be simple SEND_STRINGS()

File: keyboards/planck/keymaps/kifinnsson/keymap.c [hunk 3]
```diff
@@ -0,0 +1,584 @@
...
+        //
+        // Layout Macros
+        //
+        // Enable Colemak layout
+        case COLEMAK:
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(_COLEMAK);
+            }
+            return false;  // Skip all further processing of this key
+            break;
```
Comment (drashna) [inline keyboards/planck/keymaps/kifinnsson/keymap.c]: You only have the Colemak layer.  You can remove this:
```suggestion
```

File: keyboards/planck/keymaps/kifinnsson/keymap.c [hunk 4]
```diff
@@ -0,0 +1,584 @@
...
+     */
+    [_ADJUST] = LAYOUT_planck_grid(
+        _______, RESET,   DEBUG,   _______, _______, _______, _______, _______, _______,  _______, _______, _______,
+        _______, _______, _______, _______, _______, AG_NORM, AG_SWAP, _______, COLEMAK,  _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, TERM_ON, TERM_OFF, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______
+        )
+    };
+
+uint32_t layer_state_set_user(uint32_t state) { return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST); }
```
Comment (drashna) [inline keyboards/planck/keymaps/kifinnsson/keymap.c]:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) { return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST); }
```

File: keyboards/planck/keymaps/kifinnsson/keymap.c [hunk 5]
```diff
@@ -0,0 +1,584 @@
...
+
+bool     is_lgui_active = false;
+uint16_t lgui_timer     = 0;
+
+
+enum planck_layers { _COLEMAK, _LOWER, _RAISE, _EXTEND, _MACRO, _ADJUST };
+
+enum planck_keycodes {
+    COLEMAK = SAFE_RANGE,
+    KI_BSPC,
```
Comment (drashna) [inline keyboards/planck/keymaps/kifinnsson/keymap.c]: You only have colemak as a layout, you can remove the keycode. 

```suggestion
    KI_BSPC = SAFE_RANGE,
```

**Pull request merged**